crypto/BRCryptoWalletManagerClient.c

```diff
@@ -667,30 +670,39 @@ cwmTransactionEventAsBTC (BRWalletManagerClientContext context,
             assert (NULL != transfer);
 
             BRCryptoTransferState oldState = cryptoTransferGetState (transfer);
-            BRCryptoTransferState newState = cryptoTransferStateInit (CRYPTO_TRANSFER_STATE_CREATED);
 
-            int changed = 0;
             if (CRYPTO_TRANSFER_STATE_INCLUDED == oldState.type &&
                 (0 == event.u.updated.timestamp || TX_UNCONFIRMED == event.u.updated.blockHeight)) {
-                // newState already initialized to CRYPTO_TRANSFER_STATE_CREATED
-                changed = 1;
+                // The transfer is not included so set it to the submitted state at this point
+                BRCryptoTransferState newState = cryptoTransferStateInit (CRYPTO_TRANSFER_STATE_SUBMITTED);
+
+                cryptoTransferSetState (transfer, newState);
+
+                cwm->listener.transferEventCallback (cwm->listener.context,
+                                                     cryptoWalletManagerTake (cwm),
+                                                     cryptoWalletTake (wallet),
+                                                     cryptoTransferTake (transfer),
+                                                     (BRCryptoTransferEvent) {
+                                                         CRYPTO_TRANSFER_EVENT_CHANGED,
+                                                         { .state = { oldState, newState }}
+                                                     });
```

Comment:
I see that `oldState` safely holds `fee` (from the change to `cryptoTransferGetState()`)